package.json

```diff
@@ -178,7 +178,8 @@
     "readline-sync": "^1.4.9",
     "regenerator-runtime": "^0.13.9",
     "require-from-string": "^2.0.2",
-    "supports-hyperlinks": "^1.0.1"
+    "supports-hyperlinks": "^1.0.1",
+    "typescript": "^4.5.5"
```

Comment:
1. you need to move it to "dependencies", i.e remove from "devDependenencies" as well, or project might end up with conflicting versions (i.e somebody updates only or dependencies devDependenencies)
2. I'm not familiar how this project parses .ts files runtime, you sure it's not using ts-node?

Reply:
Yep, we're definitely not adding typescript to the dependencies - it's already the `devDependencies` section

Reply:
the opposite, **Move** from devDependencies to dependencies, not just "**Add**", if you follow the discussion.

- https://github.com/danger/danger-js/pull/1275#issuecomment-1428491746

Reply:
Done